pom.xml

```diff
@@ -67,6 +68,11 @@
     <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
     <nexus.staging.maven.plugin.version>1.6.8</nexus.staging.maven.plugin.version>
     <lifecycle.mapping.version>1.0.0</lifecycle.mapping.version>
+
+    <!-- hcom specific plugin properties -->
```

Comment:
Since this is on public Github use `hotels.com` instead of `hcom` as nobody outside the company knows our internal abbreviations.